metabase/metabase
# ECharts funnel bar chart

Closes https://github.com/metabase/metabase/issues/35824

### Description

Funnel chart has Bar chart mode in which we render all steps as bars each has own color. This PR leverages changes from https://github.com/metabase/metabase/pull/37142 to render it using the existing BarChart component.

### How to verify

1. Create a funnel chart
2. Change its mode to "Bar"
3. Ensure it renders correct funnel bar chart
4. Ensure tooltip and drills work as expected

### Demo

<img width="702" alt="Screenshot 2024-01-09 at 11 22 33 PM" src="https://github.com/metabase/metabase/assets/14301985/a1b4f468-4b9a-4b76-92e0-bd0213e72ba8">

### Checklist
